example/lambdas/s3AccessTest/index.js

```diff
@@ -38,8 +38,8 @@ async function testWrite(s3, params) {
 }
 
 /**
- * Lambda handler that tests WRITE access on an S3 bucket by calling putObject on
- * a protected bucket. Returns true if the write succeeds, false otherwise.
+ * Lambda handler that tests READ access on an S3 bucket by calling listObjectsV2 on
+ * a protected bucket. Returns true if the read succeeds, false otherwise.
```

Comment:
Not sure we're even using this file, but found this error when I was looking at it.